src/google/google-calendars/googlecalendarsyncadaptor.cpp

```diff
@@ -41,9 +43,61 @@
 #define KDATEONLY_FORMAT    "%Y-%m-%d"
 #define QDATEONLY_FORMAT    "yyyy-MM-dd"
 #define KLONGTZ_FORMAT      "%:Z"
+#define RFC5545_KDATETIME_FORMAT "%Y%m%dT%H%M%SZ"
+#define RFC5545_KDATETIME_FORMAT_NTZC "%Y%m%dT%H%M%S"
+#define RFC5545_QDATE_FORMAT "yyyyMMdd"
 
 namespace {
 
+
+void errorDumpStr(const QString &str)
+{
+    // Dump the entire string to the log.
+    // Note that the log cannot handle newlines,
+    // so we separate the string into chunks.
```

Comment:
This is a pending bug in the journal logging path. I should probably fix this instead of it being worked around seperately everywhere.